gems/aws-sdk-core/features/env.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 # WARNING ABOUT GENERATED CODE
```

Comment:
I suppose this might be dead code as well?

Reply:
I don't think so.. I'll investigate soon.

Reply:
It looks used. Let's keep it for now - if it's blown away we can investigate.